pages/common/dict.md

```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
+> More information: <https://github.com/cheusov/dictd>.
+
+- List available databases:
+
+`dict -D`
+
+- Get database information:
+
+`dict -i {{database_name}}`
+
+- Look up word in database:
+
+`dict -d {{database_name}} {{word}}`
+
+- Look up word in all available databases:
+
+`dict {{word}}`
+
+- Show server info:
```

Comment:
```suggestion
- Show information about the DICT server:
```
Not sure if `DICT` is correct, but it should be something like that

Reply:
Yes!! Thank you, @marchersimon